scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -103,6 +103,12 @@
                      'legacy behavior.\n',
                      convert_to=str, default="docker_api")
 
+define_config_option( __monitor__, 'docker_raw_logs',
+                     'Optional (defaults to False). If True, the docker monitor will use the raw log files on disk to read logs.'
```

Comment:
I would also document here how we determine the path to the log file.  Reading this, I start looking for the option on how I specify that.